packages/fxa-settings/src/pages/Pair/Index/en.ftl

```diff
@@ -0,0 +1,18 @@
+## Pair index page
+
+pair-sync-header = Sync { -brand-firefox } on your phone or tablet
+pair-cad-header = Connect { -brand-firefox } on another device
+pair-already-have-firefox-paragraph = Already have { -brand-firefox } on a phone or tablet?
+# Clicking this button initiates the pairing process, usually by directing the user to the `about:preferences` page in Firefox
+pair-sync-your-device-button = Sync your device
+# This is an h2 element immediately preceded by "Sync your device"
+pair-or-download-subheader = Or download
```

Comment:
Really great that you included the preceding context, because localizers don't have visibility on surrounding strings. Also good if we can include info about what will be downloaded since this is an incomplete  phase. 
```
# This is a heading element immediately preceded by "Sync your device" and followed by a link and QR code to download Firefox
```

Reply:
+1, this adjusted comment would be helpful.

Reply:
ooo thank you! Will fix!

Reply:
Sorry I misunderstood the initial comment and thank you for your patience!